scripts/travis/05-update-2-3.sh

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env bash
 
+# Update paragraphs to version 1.9 first.
+cd ${TEST_DIR}
+composer require drupal/paragraphs:1.3 drupal/core:~8.7.0 --no-update
+composer update
+cd ${TEST_DIR}/docroot
+drush updb -y
+cd ${TEST_DIR}
+composer require drupal/paragraphs:1.9 --no-update
+composer update
+cd ${TEST_DIR}/docroot
+drush updb -y
+# Remove paragraphs dependency.
+cd ${TEST_DIR}
+composer remove drupal/paragraphs --no-update
+composer update
+
```

Comment:
First we need 1.3. Then we need 1.9 and after that, we can update to 1.11. This is needed because paragraphs_update_8013() will fail with the latest changes of ParagraphStorageSchema if paragraphs_update_8018 and paragraphs_update_8019 are not executed. So we have to paragraphs_update_8013 without the latest changes...